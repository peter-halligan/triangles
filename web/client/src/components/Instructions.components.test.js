import React from 'react';
import ReactDOM from 'react-dom';
import { Instructions } from './Instructions.component';
import renderer from 'react-test-renderer';

test('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Instructions />, div);
  ReactDOM.unmountComponentAtNode(div);
});

test(' matches the snapshot ',  () =>{
  const tree = renderer
    .create(<Instructions />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});