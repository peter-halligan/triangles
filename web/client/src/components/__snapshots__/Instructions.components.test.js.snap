// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` matches the snapshot  1`] = `
<div>
  <div>
    A Simple application to take in triangle side lengths and return the type of triangle.
  </div>
  <div>
    
  </div>
  <div>
    You can also download a cli which contains this functionality or just use the api try using post man to hit post /triangle {a:1, b:2 c: 3} 
  </div>
</div>
`;
